### PR TITLE
Remove: Version: 0.0.0 from bower.json to enable nbt bottle

### PR DESCRIPTION
**Pull request by andygout**
cc @adgad @ironsidevsquincy

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
   "name": "n-image",
-  "version": "0.0.0",
```
Comment (ironsidevsquincy) [inline bower.json]: shouldn't this be set to the version?

Comment (andygout) [inline bower.json]: Am getting this error when running `nbt bottle` while listing a version:-

`bower.json should not contain a version property; versioning is done with git tags`

Have looked at other bower components (all of those required by `next-front-page`) and none of the `bower.json` files for those contain a version number.

Comment (matthew-andrews) [inline bower.json]: @ironsidevsquincy the version number in bower.json is completely optional, bower only really uses git tags.

**Comment by ironsidevsquincy** [comment]
:+1:

**Pull request merged**